js/src/mpl.js

```diff
@@ -160,7 +157,7 @@ mpl.figure.prototype._init_canvas = function() {
 
     var pass_mouse_events = true;
 
-    canvas_div.resizable({
+    /*canvas_div.resizable({
```

Comment:
(which will be merged into the main repo once we figure out https://github.com/phosphorjs/phosphor/issues/213